src/main/java/net/mcreator/ui/workspace/resources/TextureType.java

```diff
@@ -31,4 +33,18 @@
 	public String getID() {
 		return id;
 	}
+
+	/**
+	 * <p>Get the {@link TextureType} in the enum depending on {@param position} and {@param withArmor}.</p>
+	 *
+	 * @param position <p>The position of the texture type in the enum.</p>
+	 * @param withArmor <p>Set to true to include the armor texture type into the possible types.</p>
+	 * @return <p>The corresponding {@link TextureType}</p>
+	 */
+	public static TextureType getTextureType(int position, boolean withArmor) {

```

Comment:
Enum should also return arrays with and without armor so this method getTextureType makes sense

Reply:
There is a method on `java.lang.Enum` for "with armor" cases - `values()`. We only need `valuesNoArmor()`.